src/pudl/metadata/classes.py

```diff
@@ -933,9 +933,24 @@ def raw_datapackage_title(self) -> str:
         """Construct a datapackage title for the raw data source."""
         return f"PUDL Raw f{self.title}"
 
-    def to_raw_datapackage_json(self) -> pydantic.Json:
+    def raw_datapackage_sources(self) -> List[Dict[str, str]]:
+        """Construct datapackage sources for raw data source."""
+        return [{"title": self.title, "name": self.path}]
+
+    def to_raw_datapackage_dict(self) -> Dict:
```

Comment:
I changed this to return a dict because it integrates a bit cleaner into the `pudl-zenodo-storage` code, but we could return JSON directly if that'd be preferred.

Reply:
That makes sense.

One thing that has been a little bit frustrating with the Zenodo storage setup is that there's no way to preview the `datapackage.json` file that it's going to output without actually going to the trouble of uploading all of the data to Zenodo. If it was easy to have a "dry run" or "no op" version of the command that output that JSON file or printed it out that could be helpful.